swift/modules/SwiftUtilities/Tests/SwiftUtilitiesTests/UniqueNameStringTests.swift

```diff
@@ -0,0 +1,79 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import XCTest
+@testable import SwiftUtilities
+
+final class UniqueNameStringTests: XCTestCase {
+    /// Test to ensure that calling uniqueName() with no
+    /// inputs simply returns a lowercased UUID.
+    func testSimpleUniqueName() {
+        let ts = String.uniqueName()
+        XCTAssertTrue(ts.lowercased() == ts)
+
+        let tsUUID = UUID(uuidString: String.uniqueName())
+        XCTAssertNotNil(tsUUID)
+    }
+
+    /// Ensure that the `withPrefix` property works by
+    /// checking that the returned value:
```

Comment:
/// Confirm that the `withPrefix` property works by
    /// checking that the returned value has the following properties: